.ebextensions/env.config.example

```diff
@@ -0,0 +1,6 @@
+option_settings:
```

Comment:
What's the difference between this file and our current `.env.local`?

Reply:
It holds the same info, but this one is stored in EB, so we don't have to keep track of credentials locally and could make updates to them in the AWS console without redeploying.